### ISSUE TITLE
bug: Accidentally addad files breaks clone in Windows

### ISSUE THREAD

**Issue by mg-bayoo** [EdenEast/nightfox.nvim#339]
### Neovim version (nvim -v)

0.9.0

### Operating system/version

Windows

### Describe the bug

In the last PR (#338) a file was added to the root directory with the same file name as the commit message.

This breaks the cloning in Windows with the following Error:
![image](https://user-images.githubusercontent.com/104770719/235143126-6ecfcbbb-7f75-43c6-b546-8654c811542b.png)

### Steps To Reproduce

1. Use Windows
2. Try to clone the repository (or use your favorite Plugin Manager to update the plugin)

### Expected Behavior

Clone should work

### Repro

```Lua
-- DO NOT change the paths and don't remove the colorscheme
local root = vim.fn.fnamemodify("./.repro", ":p")

-- set stdpaths to use .repro
for _, name in ipairs({ "config", "data", "state", "cache" }) do
  vim.env[("XDG_%s_HOME"):format(name:upper())] = root .. "/" .. name
end

-- bootstrap lazy
local lazypath = root .. "/plugins/lazy.nvim"
if not vim.loop.fs_stat(lazypath) then
  vim.fn.system({ "git", "clone", "--filter=blob:none", "https://github.com/folke/lazy.nvim.git", lazypath })
end
vim.opt.runtimepath:prepend(lazypath)

-- install plugins
local plugins = {
  {
    "EdenEast/nightfox.nvim",
    config = function()
      require("nightfox").setup({
        -- setup here ...
      })
    end,
  },
  -- add any other plugins here
}
require("lazy").setup(plugins, {
  root = root .. "/plugins",
})

vim.cmd.colorscheme("nightfox")
```

**Comment by linrongbin16** [comment]
It looks like a git commit error, guess the author submit commit message as a text file.

**Comment by EdenEast** [comment]
Resolved by #340
